hedera-node/configuration/mainnet/settings.txt

```diff
@@ -30,6 +30,7 @@ event.enableEventStreaming,                    true
 event.eventsLogDir,                            /opt/hgcapp/eventsStreams
 event.eventsLogPeriod,                         5
 event.maxEventQueueForCons,                    1000
+event.creation.maxCreationRate,                1
```

Comment:
The `maxCreationRate` for mainnet shouldn't be `1`.

IMO, you should only change the value for whichever config file manages the integration tests in the PR.

It still remains to be seen what a reasonable value will be for mainnet and other networks

Reply:
Thanks @alittley . Removed from mainnet and testnet settings